rllib/algorithms/alpha_zero/alpha_zero.py

```diff
@@ -109,7 +109,7 @@ def __init__(self, algo_class=None):
         self.replay_buffer_config = {
             "type": "ReplayBuffer",
             # Size of the replay buffer in batches (not timesteps!).
-            "capacity": 1000,
+            "capacity_items": 1000,
```

Comment:
capacity is gone and replaced by capacity_ts, capacity_items and capacity_bytes.